staging/src/k8s.io/apiserver/pkg/storage/etcd3/watcher.go

```diff
@@ -87,34 +86,18 @@ type watchChan struct {
 	errChan           chan error
 }
 
-func newWatcher(client *clientv3.Client, codec runtime.Codec, groupResource schema.GroupResource, newFunc func() runtime.Object, versioner storage.Versioner) *watcher {
```

Comment:
I'm planing to add more fields to the `watcher` struct in https://github.com/kubernetes/kubernetes/pull/119557.